api_types.ts

```diff
@@ -761,6 +761,12 @@ export interface SyncExecutionContext extends ExecutionContext {
   readonly sync: Sync;
 }
 
+export interface CellAutocompleteExecutionContext extends ExecutionContext {
+  readonly propertyName: string;
```

Comment:
This is the name of the field that corresponds to the cell being edited?

Reply:
Yes, this indicates the column being edited.

Reply:
Updated https://github.com/coda/coda/pull/86884/files#diff-5ac16ad7e4797378dc795e09c63d69f986464ac39306c7d7d622c0738b6a4b7a so the keys are untransformed before calling calc service